tools/cli/src/mentat_cli/command_parser.rs

```diff
@@ -207,6 +236,13 @@ mod tests {
     }
 
     #[test]
+    fn test_open_parser_no_args_trailing_whitespace() {
+        let input = ".open ";
+        let err = command(&input).expect_err("Expected an error");
```

Comment:
I see that `expect_err` is in 1.17.0 stable, so we can use it -- but we need to bump https://github.com/mozilla/mentat/blob/8a8fcedd1c9d57ed6673f398b84bf72aa9f0bd5f/build/version.rs#L19 as a Pre: part to stop folks failing here (as I just did locally!).